docs_source_files/content/guidelines_and_recommendations/page_object_models.de.md

```diff
@@ -65,65 +67,94 @@ Applying the page object techniques, this example could be rewritten like this
 in the following example of a page object for a Sign-in page.
 
 ```java
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
 /**
  * Page Object encapsulates the Sign-in page.
  */
 public class SignInPage {
-
-        private Selenium selenium;
-
-        public SignInPage(Selenium selenium) {
-                this.selenium = selenium;
-                if(!selenium.getTitle().equals("Sign in page")) {
-                        throw new IllegalStateException("This is not sign in page, current page is: "
-                                        +selenium.getLocation());
-                }
-        }
-
-        /**
-         * Login as valid user
-         *
-         * @param userName
-         * @param password
-         * @return HomePage object
-         */
-        public HomePage loginValidUser(String userName, String password) {
-                selenium.type("usernamefield", userName);
-                selenium.type("passwordfield", password);
-                selenium.click("sign-in");
-                selenium.waitForPageToLoad("waitPeriod");
-
-                return new HomePage(selenium);
-        }
+  protected static WebDriver driver;
+
+  public SignInPage(WebDriver driver){
+    this.driver = driver;
+    PageFactory.initElements(driver, this);
```

Comment:
Could you please remove the use of `PageFactory`? A normal approach declaring locators and using them in methods should be ok.
At the Selenium project, we are encouraging people to stop using `PageFactory` since it creates sometimes several `StaleElementException`, which is kind of hard to debug when you are starting with Selenium.

Reply:
@diemol Thank you for the review! 
I've corrected what you pointed out, removing PageFactory class.